internal/cli/tiltfile_result.go

```diff
@@ -0,0 +1,88 @@
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	tiltanalytics "github.com/windmilleng/tilt/internal/analytics"
+	"github.com/windmilleng/tilt/internal/tiltfile"
+	"github.com/windmilleng/tilt/pkg/logger"
+	"github.com/windmilleng/tilt/pkg/model"
+)
+
+const TiltfileErrExitCode = 5
+
+type tiltfileResultDeps struct {
```

Comment:
it might be worth calling this cmdTiltfileResultDeps, for consistency with cmdUpDeps. it's a dependency of the command, not the result